CONTRIBUTING.md

```diff
@@ -2,13 +2,81 @@
 
 We'd love you to contribute to dyrector.io platform. We want contribution to dyrector.io to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
 
+## Code of Conduct
+
+Help us keep dyrector.io open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
 ## How to Start?
 
 If you're worried or don’t know where to start, check out our next section explaining what kind of help we could use and where you can become involved. You can reach out with questions to our [Discord server](https://discord.gg/hMyT9cbYFD) or [@dyrectorio](https://twitter.com/dyrectorio) on Twitter.
 
-## Code of Conduct
-
-Help us keep dyrector.io open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+## Development
```

Comment:
Don't want to add this to DEVELOPMENT.md?

Reply:
contributing != development?

Reply:
idk 🤡